ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/result-sort/dropdown.result-sort.view.js

```diff
@@ -13,18 +13,27 @@
  *
  **/
 /*global define*/
+import * as React from 'react'
```

Comment:
You only need this syntax as a workaround for tsx files. `Import React from 'react'` will pull in less of that dependency.

Reply:
Actually on second thought, you might as well make this a `tsx` file.

Reply:
I haven't seen any marionette code using typescript, we might be only doing for react components

Reply:
changed import statement